code/modules/reagents/chemistry/reagents/toxins.dm

```diff
@@ -176,6 +176,38 @@
 	return ..()
 
 
+/datum/reagent/stable_mutagen
+	name = "Stable mutagen"
+	id = "stable_mutagen"
+	description = "Just the regular, boring sort of mutagenic compound.  Works in a completely predictable manner."
+	reagent_state = LIQUID
+	color = "#7DFF00"
```

Comment:
Missing rgb comment. `rgb(125, 255, 0)`

Reply:
also according to your mix message, I wouldn't call neon green 'drab', maybe change the color? Just a thought homeslice.